templates/vpc-ec2-rds-alb.yaml

```
@@ -0,0 +1,340 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Create VPC with public & private subnets, route tables, security groups, a t2.micro EC2, Internet-facing ALB (HTTP → 8080) and MySQL RDS
+
+Parameters:
+  KeyName:
+    Type: AWS::EC2::KeyPair::KeyName
+    Description: "Name of an existing EC2 KeyPair to enable SSH access to the instance"
+    Default: "key-test"
+
+  AmiId:
+    Type: AWS::EC2::Image::Id
+    Description: AMI for Amazon Linux 2023 (x86_64)
+    Default: ami-05206bf8aecfc7ae6
+
+  InstanceType:
+    Type: String
+    Default: t2.micro
+
+  DbUser:
+    Type: String
+    Default: root
```

```suggestion
  DbUser:
    Type: String
    Default: root
    Description: "RDS master user name"
```
パスワードの定義に合わせるならこんな感じですかね！